source/mir/random/engine/pcg.d

```diff
@@ -248,8 +249,8 @@ public:
         /// ditto
         void seed()(Uint seed) { this.__ctor(seed); }
         /// ditto
-        static if (streamType == stream_t.none || streamType == stream_t.oneseq)
```

Comment:
This needs to be preserved for `hasMember` traits, otherwise, it will return true while it should return false.

Reply:
Fixed.